test/functional/sim_coinselect.py

```diff
@@ -111,7 +111,7 @@ def log_sim_results(self, res_file):
 
         # Remaining utxos and fee stats
         remaining_utxos = self.tester.listunspent()
-        cost_to_empty = Decimal(-1) * Decimal(len(remaining_utxos)) * Decimal(68) * Decimal(0.00001) / Decimal(1000)
+        cost_to_empty = Decimal(1) * Decimal(len(remaining_utxos)) * Decimal(68) * Decimal(0.0001) / Decimal(1000)
```

Comment:
You could just remove `Decimal(1)`. Multiplying by 1 is unnecessary.

Reply:
Okay, I wasn't sure whether it was needed here, given that there were other multiplications with Decimal(1) on other lines surrounding this. Will remove.